packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -304,7 +327,8 @@ const BookerPlatformWrapperComponent = (
       Boolean(timezone) &&
       !event?.isPending &&
       event?.data?.id != null,
-    orgSlug: props.entity?.orgSlug ?? undefined,
+    // Use resolved orgSlug for availability - falls back gracefully if not available
```

Comment:
Can we remove all comments to files added in this PR except documentation? This comment simply restates the code and if the code updates then we also need to maintain this doc, so its easier to delete it.